bundles/org.openhab.binding.miio/README.md

```diff
@@ -6517,6 +6541,32 @@ String SerialNumber "Device Information-Device Serial Number" (G_plug) {channel=
 Switch On "Switch-Switch Status" (G_plug) {channel="miio:basic:plug:On"}
 ```
 
+### Xiaomi Smart Plug 2 (Wi-Fi) (cuco.plug.v2eur) item file lines
+
+note: Autogenerated example. Replace the id (plug) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
+
+```
```

Comment:
```suggestion
```java
```

Reply:
@fwolter asked me to remove that in earlier review.

https://github.com/openhab/openhab-addons/pull/9968/files/f84cbdd24c910e0f57f97805ee7329afbb736dfd#r564631582

Reply:
That was 3 years ago, back there was no convention yet, bit since https://github.com/openhab/openhab-docs/issues/999 there is. Almost all bindings have been corrected last year.